src/crafting_gui.cpp

```diff
@@ -512,23 +517,29 @@ const recipe *select_crafting_recipe( int &batch_size )
                                    current[line]->has_flag( flag_BLIND_EASY ) ? _( "Easy" ) :
                                    current[line]->has_flag( flag_BLIND_HARD ) ? _( "Hard" ) :
                                    _( "Impossible" ) ) );
-                if( available[line].can_craft && !available[line].can_craft_non_rotten ) {
+                const bool can_craft_this = available[line].can_craft;
+                if( can_craft_this && !available[line].can_craft_non_rotten ) {
                     ypos += fold_and_print( w_data, point( xpos, ypos ), pane, col,
                                             _( "<color_red>Will use rotten ingredients</color>" ) );
                 }
                 const bool too_complex = current[line]->deduped_requirements().is_too_complex();
-                if( available[line].can_craft && too_complex ) {
+                if( can_craft_this && too_complex ) {
                     ypos += fold_and_print( w_data, point( xpos, ypos ), pane, col,
                                             _( "Due to the complex overlapping requirements, this "
                                                "recipe <color_yellow>may appear to be craftable "
                                                "when it is not</color>." ) );
                 }
-                if( !available[line].can_craft && available[line].apparently_craftable ) {
+                if( !can_craft_this && available[line].apparently_craftable && available[line].has_proficiencies ) {
                     ypos += fold_and_print(
                                 w_data, point( xpos, ypos ), pane, col,
                                 _( "<color_red>Cannot be crafted because the same item is needed "
                                    "for multiple components</color>" ) );
                 }
+                if( !can_craft_this && !available[line].has_proficiencies ) {
+                    ypos += fold_and_print( w_data, point( xpos, ypos ), pane, col,
+                                            _( "<color_red>Cannot be crafted because you lack"
+                                               " the required proficiencies.</color>" ) );
+                }
```

Comment:
I feel like there should be something in the crafting UI about how much slower this recipe is because of proficiencies you lack.  I didn't see that.

Reply:
It's in formatting the proficiency string in the recipe screen (the time multiplier next to recipes is yellow if it applies, white if it does not), but I agree, it's not very clear, either in the UI or code. I'll add a message here about it.